emperor/support_files/js/model.js

```diff
@@ -276,6 +296,35 @@ function ($, _) {
 
   /**
    *
+   * Helper function used to find the minimum and maximum values every
+   * dimension in the plottable objects. This function is used with
+   * uderscore.js' reduce function (_.reduce).
+   *
+   * @param {accumulator} Object with a "min" and "max" arrays that store the
+   * minimum and maximum values over all the plottables.
+   * @param {plottable} Plottable object to compare with.
+   *
+   * @return Updated version of accumulator, integrating the ranges of the
+   * newly seen plottable object.
+   *
+   **/
+  DecompositionModel._minMaxReduce = function(accumulator, plottable){
+
+    // iterate over every dimension
+    _.each(plottable.coordinates, function(value, index){
```

Comment:
I don't think we can use `min` here, because we don't have access to vectorized operations over the whole matrix of coordinates, that's why we iterate over each one of them. I realize this is a bit confusing, but `_.reduce` is every plottable and for every plottable it finds the minimum and maximum over each axis, that's why we use `_.each` here.